### ISSUE TITLE
Provide users with more sign in/sign up options

### ISSUE THREAD

**Issue by RohankumarM** [codeforcauseorg/edu-client#209]
Provide users with more options to sign in/sign up like Facebook, Yahoo, and Github.

Different buttons could be designed to handle each case.

**Comment by RohankumarM** [comment]
Hey @Abhishek-kumar09, is this feature needed? If yes, I could start working on it?

**Comment by Abhishek-kumar09** [comment]
Yes, For now we have a google signIn only, we could also give option to github signIn.
If you want you can work on it.

Btw, the changes are needed to be tested in android application by placing service files from firebase to android folder at app level I guess.

**Comment by RohankumarM** [comment]
@Abhishek-kumar09 @Abhijay007 Could you let me know if the sign-in method using GitHub and Facebook is enabled in firebase authentication?

**Comment by Abhijay007** [comment]
> @Abhishek-kumar09 @Abhijay007 Could you let me know if the sign-in method using GitHub and Facebook is enabled in firebase authentication?

@RohankumarM we will let you know about it when we gonna implement it, we are currently planning to add an OTP option as well and making a new UI design for the authentication page because of which this issue is on hold for now.

**Comment by RohankumarM** [comment]
> > @Abhishek-kumar09 @Abhijay007 Could you let me know if the sign-in method using GitHub and Facebook is enabled in firebase authentication?
> 
> @RohankumarM we will let you know about it when we gonna implement it, we are currently planning to add an OTP option as well and making a new UI design for the authentication page that's why this issue is on hold for now.
That sounds great. Even I was thinking of OTP feature. When will I hear from you?

**Comment by Abhijay007** [comment]
> That sounds great. Even I was thinking of OTP feature. When will I hear from you?

@RohankumarM I don't know much about the implementation plans you may need to ask mentors about this.
